sdk/spring/azure-spring-cloud-servicebus-stream-binder-core/src/main/java/com/azure/servicebus/stream/binder/properties/package-info.java

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+/**
+ * Package com.azure.servicebus.stream.binder.properties
+ */
+package com.azure.servicebus.stream.binder.properties;
```

Comment:
Let's change the package name to com.azure.spring.xxxx

Reply:
done